addons/mrp/models/stock_picking.py

```diff
@@ -53,6 +53,13 @@ def get_mrp_stock_picking_action_picking_type(self):
 class StockPicking(models.Model):
     _inherit = 'stock.picking'
 
+    has_kits = fields.Boolean(compute='_compute_has_kits')
+
+    @api.depends('move_lines')
+    def _compute_has_kits(self):
+        for picking in self:
+            picking.has_kits = any(picking.move_lines.mapped('description_bom_line'))
```

Comment:
bom_line_id does the same no? With a smaller computation

Reply:
I think you are right. Do you think it's safer to check the bom type? Although all the bom line should come from a kit bom now.

Reply:
Nah just check if bom_line_id is set